package.json

```diff
@@ -149,7 +149,7 @@
     "Other"
   ],
   "engines": {
-    "vscode": "^1.87.0"
+    "vscode": "^1.97.0"
```

Comment:
Mostly unrelated change to avoid this error:

```
~/code/vscode-open-in-github  remove-trailing-slash-from-permalink ✔ 1↥  06:01:10 PM
❯ vsce package
 ERROR  @types/vscode ^1.97.0 greater than engines.vscode ^1.87.0. Either upgrade engines.vscode or use an older @types/vscode version
```

Follow-up to https://github.com/davidrunger/vscode-open-in-github/pull/5/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R166 .